samples/batch/csharp/README.md

```diff
@@ -1,6 +1,6 @@
-**C# Example to use the Speech Services Batch Transcription API**
+# C# Example to use the Speech Services Batch Transcription API
 
-Speech Service Batch Transcription is exposed through a REST API. The samples here does **NOT** require the installation of the Cognitive Service Speech SDK, but uses the REST API directly.
+Speech Services Batch Transcription is exposed through a REST API. The samples here does **NOT** require the installation of the Cognitive Service Speech SDK, but uses the REST API directly.
```

Comment:
typo: The sample here does NOT .. should be do NOT

Reply:
also uses -> use

Reply:
fixed, thanks